src/common.js

```diff
@@ -6,16 +6,10 @@ Common.prototype.exampleMethod = function () {
 }
 
 Common.prototype.logId5Id = function (id5Id) {
-    //Checks and saves ID5 ID if it is new
-    if (id5Id !== this.id5Id) {
-        this.id5Id = id5Id;
-        this.id5IdSent = false
-    }
-
-    //Sets user attribute if ID is unsent.
-    //This function will be updated once the decryption architecture is finalized.
-    //TO-DO: Log the ID5 ID to correct location
-
+    var integrationAttributes = {};
+    integrationAttributes['encryptedId5Id'] = id5Id;
+    integrationAttributes['id5IdType'] = this.id5IdType;
```

Comment:
Some of these keys like `Id5Id` look like it would be very prone to typos so I would make them constants. However, it might be cleaner to do it this way:

```suggestion
    var integrationAttributes = {
        encryptedId5Id: id5Id,
        id5IdType: this.id5IdType
    };
```

Although I would be curious if there's any issue with either of these values being `null` or `undefined`. If that's the case, I would come up with default values.

Reply:
`id5Id` is already checked for null or undefined above in the function. The id5IdType is a required setting for the integration so should always be present.